docs/layouts/shortcodes/generated/common_high_availability_jrs_section.html

```diff
@@ -0,0 +1,24 @@
+<table class="configuration table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">Key</th>
+            <th class="text-left" style="width: 15%">Default</th>
+            <th class="text-left" style="width: 10%">Type</th>
+            <th class="text-left" style="width: 55%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>job-result-store.delete-on-commit</h5></td>
+            <td style="word-wrap: break-word;">true</td>
+            <td>Boolean</td>
+            <td>Determines whether job results should be automatically removed from the underlying job result store when they are clean. If false, the cleaned job results are, instead, marked as clean to indicate their state. In this case, Flink no longer hasownership and the resources need to be cleaned up by theuser.</td>
+        </tr>
+        <tr>
+            <td><h5>job-result-store.storage-path</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
+            <td>String</td>
+            <td>Defines where job results should be stored. This should be an underlying file-system that provides read-after-write consistency. By default, this is <code class="highlighter-rouge">high-availability.storageDir/job-result-store/high-availability.cluster-id</code>.</td>
```

Comment:
Yeah, this is much clearer in the resulting documentation. I'll change this as suggested in the Option itself, but I think it might be a worthwhile extension to the `org.apache.flink.configuration.description.DescriptionBuilder`.